module/applications/item/item-sheet.mjs

```diff
@@ -110,7 +110,7 @@ export default class ItemSheet5e extends ItemSheet {
     /** @deprecated */
     Object.defineProperty(context, "data", {
       get() {
-        const msg = `You are accessing the "data" attribute within the rendering context provided by the ItemSheet5e 
+        const msg = `You are accessing the "data" attribute within the rendering context provided by the ItemSheet5e
```

Comment:
I actually wonder why this is a template string at all. I thought our linter would flag these sorts of usages. Not relevant to this review though.